src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      class="form"
+      method="post"
+      data-qa="big"
+      >
+      <label class="search">
+        <img
+          class="icon-big"
+          src="./images/Search.svg"
+           alt="search-big"
+           >
```

Comment:
```suggestion
        <img
          class="icon-big"
          src="./images/Search.svg"
          alt="search-big"
        >
```
Recheck formatting style guides and fix it everywhere
https://mate-academy.github.io/style-guides/htmlcss.html